### PR TITLE
fix(a11y): Fix informative graphics contrast of user status icons

### PR DESCRIPTION
**Pull request by Pytal**
- For https://github.com/nextcloud/server/issues/42644

- Requires https://github.com/nextcloud-libraries/nextcloud-vue/pull/5059

> Opening for review in advance

## Summary

Before | After
--- | ---
image | image
image | image
image | image
image | image

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/user_status/src/UserStatus.vue [hunk 1]
```diff
@@ -25,16 +25,25 @@
 		<button v-if="!inline"
 			class="user-status-menu-item"
 			@click.stop="openModal">
-			<span aria-hidden="true" :class="statusIcon" class="user-status-icon" />
+			<NcUserStatusIcon class="user-status-icon"
+				:class="{
+					'user-status-icon--invert': statusType === 'invisible',
+				}"
```
Comment (ShGKme) [inline apps/user_status/src/UserStatus.vue:31]: Cannot inverting also be covered by the component?

**Review by ShGKme — APPROVED** [review]

File: apps/dashboard/src/DashboardApp.vue [hunk 2]
```diff
@@ -74,7 +74,8 @@
 							:checked="isStatusActive(status)"
 							@input="updateStatusCheckbox(status, $event.target.checked)">
 						<label :for="'status-checkbox-' + status">
-							<span :class="statusInfo[status].icon" aria-hidden="true" />
+							<NcUserStatusIcon v-if="status === 'status'" status="online" aria-hidden="true" />
```
Comment (emoral435) [inline apps/dashboard/src/DashboardApp.vue:77]: as status is capitalized, shouldn't the conditional rendering be
`v-if="status === 'Status'"` ?

Comment (Pytal) [reply apps/dashboard/src/DashboardApp.vue:77]: The `status` here is matching the instance registered with the key `'status'` https://github.com/nextcloud/server/blob/587057b0edf4e3b17772cdcc686021ea6d6c938b/apps/user_status/src/menu.js#L63

**Review by susnux — APPROVED** [review]

**Review by emoral435 — APPROVED** [review]

**Pull request merged**